src/preload.py

```diff
@@ -956,6 +956,12 @@ def get_partition(name, image=None):
     for line in sys.stdin:
         data = json.loads(line)
         method = methods[data["command"]]
-        result = method(**data.get("parameters", {}))
+        try:
+            result = method(**data.get("parameters", {}))
+        except BaseException as e:
+            result = {
+                "StatusCode": 1,
```

Comment:
I don't really get the point of `StatusCode`.
It will never be anything else than `1`.

`StatusCode` comes from dockerode's container.wait(), we shouldn't reuse this name